Simplified/MyBooks/NYPLMyBooksDownloadCenter.m

```diff
@@ -392,8 +400,15 @@ - (void)URLSession:(__attribute__((unused)) NSURLSession *)session
      setState:NYPLBookStateDownloadFailed
      forIdentifier:book.identifier];
   }
-
+  
+#if FEATURE_AUDIOBOOKS
+  if (book.defaultBookContentType == NYPLBookContentTypeAudiobook) {
+    [self downloadAudiobookForBook:book];
+    return;
+  }
+#else
```

Comment:
why don't we do the broadcast if we have feature audiobooks enabled?

Reply:
We will do the broadcast once the audiobook download object is created and added to the downloader.

Reply:
oh I see. Makes sense, I had not seen that.